README.md

```diff
@@ -5,26 +5,37 @@
 
```

Comment:
Maybe we can add the travis badge

`[![Build Status](https://travis-ci.org/arbolista-dev/cc-user-api.svg?branch=master)](https://travis-ci.org/arbolista-dev/cc-user-api)`